spec/helpers/ckeditor_spec.rb

```diff
@@ -0,0 +1,7 @@
+def fill_in_ckeditor(id, with:)
+
+  within_frame find("#cke_#{id} iframe") do
+    find('body').base.send_keys with
+  end
+
+end
```

Comment:
Note that `spec/helpers/` is actually for specs that test view helpers. Helper methods used by spec files (like this one) should go in `spec/support/common_actions/`.

Reply:
Done https://github.com/consul/consul/pull/2914/commits/549c08657432624654ac2d6c76beae0231d25568